recipes/example/meta.yaml

```diff
@@ -1,78 +0,0 @@
-# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```

Comment:
Please don't delete this. Can you rebase on top of master?

Reply:
I did a rebase using TurtoiseCI - not sure if it worked out the way you wanted

(thank you for baring with me)

Reply:
I've fixed it now